<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectCleanerForm</class>
 <widget class="QWidget" name="ConnectCleanerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>623</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Choose Electrodes to Keep</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="SelectElectrodeList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="MonoCheckBox">
         <property name="text">
          <string>Monopoles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="BipoleCheckBox">
         <property name="text">
          <string>Bipoles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="ValidateButton">
     <property name="text">
      <string>Validate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ExportButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
